spiffworkflow-backend/src/spiffworkflow_backend/routes/tasks_controller.py

```diff
@@ -925,9 +925,15 @@ def _task_submit_shared(
                 human_task=human_task,
             )
 
+    # currently task_model has the potential to be None. This should be removable once
+    # we backfill the human_task table for task_guid and make that column not nullable
+    task_model: TaskModel | None = human_task.task_model
+    if task_model is None:
+        task_model = TaskModel.query.filter_by(guid=human_task.task_id).first()
+
     # delete draft data when we submit a task to ensure cycling back to the task contains the
     # most up-to-date data
-    task_draft_data = TaskService.task_draft_data_from_task_model(human_task.task_model)
+    task_draft_data = TaskService.task_draft_data_from_task_model(task_model)
```

Comment:
what do you mean by "given the context"? human_task.task_model can be None, because it's a relationship based on human_task.task_guid, which is a nullable db column.